### DISCUSSION TITLE
Welcome to Unifi-Toolbox Discussions!

### DISCUSSION THREAD

**Discussion by elueckel** [elueckel/Unifi-Toolbox#35] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Brovning** [comment]
So, der Grundstein ist gelegt!
Jetzt müssen nur noch die User ihre Probleme und Fragen hier posten und nicht bei IPS. ;-)

**Comment by Brovning** [comment]
Wollen wir "Projects" entfernen und die dort eingetragenen Ideen lieber hier unter Ideas führen?

[the remark this answers is not in the document]
**Reply by Brovning** [reply]
> Ja - wäre vermutlich besser / einfacher. 

Habe alles von "Projects" in "Discussions" übernommen.
Du kannst "Projects" deaktivieren.

**Comment by methusalix2000** [comment]
Hallo Zusammen,
ich hätte eine Frage. Wäre es möglich mit diesem Tool auch einzelne PoE Ports zu aktivieren und deaktivieren?

Vielen Dank
SG, Stefan

**Reply by elueckel** [reply]
Aktuell nicht - was willst du genau machen? Was ist das "use case"

**Comment by methusalix2000** [comment]
Ich habe an den POE - W-LAN APs angeschlossen. Ich möchte gerne im Schlafzimmer diesen aber in der Nacht deaktivieren/Stromlos schalten und in der früh wieder einschalten.

**Reply by elueckel** [reply]
Ich denke das geht nicht - also mit PowerCylce auf jeden Fall nicht.

Hier sind alle Funktionen: https://ubntwiki.com/products/software/unifi-controller/api

**Reply by methusalix2000** [reply]
Ja mit PowerCycle nicht... anscheinend geht es irgendwie mit einer PortConfig .... aber da bin ich zu schwach um das herauszufinden... 
https://community.ui.com/questions/API-Call-to-Change-PoE-state/355fe447-c21e-4fe0-8e64-74f7736934ec

Werde einen POE injektor mit Relais wegschalten

Trotzdem vielen Dank
schöne grüße
